src/main/resources/templates/db_scripts/create-assigned-users-table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE assigned_users
+DROP COLUMN IF EXISTS jsonb,
+ADD COLUMN IF NOT EXISTS credentials_id uuid NOT NULL constraint fk_assigned_users_kb_credentials references kb_credentials,
```

Comment:
Does this type of constraint declaration create new constraint each time DB is created?